test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/MalformedPageDirective_DesignTime.codegen.cs

```diff
@@ -13,10 +13,6 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_MalformedP
     {
         #pragma warning disable 219
         private void __RazorDirectiveTokenHelpers__() {
-        ((System.Action)(() => {
-global::System.Object __typeHelper = "foo;
```

Comment:
What leads to this being omitted? I agree that it's good, but I want to understand why.